cloud_functions/token_refresh/main.py

```
@@ -0,0 +1,38 @@
+"""Main entry point for cloud function."""
+
+import functions_framework
+from google.cloud import firestore
+from streams.strava import refresh_oauth_token
+from utils.logger_utils import logger
+
+
+@functions_framework.http
+def refresh_token(request):
```

Ruff (0.8.2)

Missing return type annotation for public function `refresh_token`

(ANN201)